packages/design-system/src/components/button/stories/index.js

```diff
@@ -213,7 +197,7 @@ ToggleButtonContainer.propTypes = {
   type: PropTypes.oneOf(Object.values(BUTTON_TYPES)),
 };
 
-export const ToggleButtons = () => {
+export const ToggleButtons = (args) => {
```

Comment:
🤔 Seems like this story isn't configured to use controls?
![image](https://user-images.githubusercontent.com/22185279/151239670-ee2bed29-aa38-48d9-9a3a-7f6951d5bb71.png)

Reply:
actions will be done separately to try and not add more to this already large PR: https://github.com/GoogleForCreators/web-stories-wp/issues/10375

Reply:
Correct. With the toggle button there isn't anything to `control`. Here is the weird it, the `args` are passed in solely to be able to filter out all the `controls`. via line 229 `ToggleButtons.parameters = { controls: { include: [] } };`. This makes it so that `variant` `size` and `type` aren't shown as a control, since they do nothing on these toggle buttons.